### ISSUE TITLE
[Feature Request] Application borderless window as desktop background

### ISSUE THREAD

**Issue by KSXGitHub** [Komorebi-Fork/komorebi#33]
I sometimes want a semi-transparent system monitor (such as htop) on top of a background image as my desktop wallpaper. This feature would also greatly expand customisability too.

**Comment by Ev1lbl0w** [comment]
Hi, I assume you're requesting to be able to make an open application an "wallpaper", right?

I doubt this will be an easy feat to integrate. However, you might be able to use [xwinwrap](https://github.com/ujjwal96/xwinwrap) to accomplish just that.

**Comment by KSXGitHub** [comment]
> I assume you're requesting to be able to make an open application an "wallpaper"

That is correct. However, the application window should appear semi-transparent, enable user to see through the background image behind.

> you might be able to use [xwinwrap](https://github.com/ujjwal96/xwinwrap) to accomplish just that

Thanks for your guidance. I will try it.

**Comment by KSXGitHub** [comment]
@Ev1lbl0w I don't know how to use xwinwrap. I tried executing `xwinwrap -fs -- alacritty -e htop` but it opens 2 windows (one is all black, the other is htop on alacritty) and none of them is in background.

**Comment by Ev1lbl0w** [comment]
Sorry, but can't help you much, I never used this tool. Your best bet would be to open up a new issue there so they can help you achieve what you want.
